ui.apps/src/main/content/jcr_root/apps/authoring-toolkit/custom-lists/components/content/console/.content.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This one's great. and I'd add a licence here because this is much of a creative work